src/SDKs/GuestConfiguration/Management.GuestConfiguration/Microsoft.Azure.Management.GuestConfiguration.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.GuestConfiguration</PackageId>
+    <Description>Provides Microsoft Azure GuestConfiguration management operations including the ability to create, update and get guest configuration assignments, compliance statuses, guest configuration assignment reports. </Description>
+    <Version>1.0.0.0</Version>
```

Comment:
If the REST API is in preview the Nuget package must also be preview.
We recommend versioning should start from `0.9.0-preview`
Please update the `AssemblyInfo.cs`  accordingly

Reply:
Updated.  in AssemblyInfo.cs as well. Please check